lib/browser/api/extensions.js

```diff
@@ -259,6 +259,25 @@ const createTab = function (createProperties, cb) {
   }
 }
 
+const duplicateTab = function (tabId, cb) {
+  try {
+    let tab = getWebContentsForTab(tabId)
+    // XXX What to do if not there?
```

Comment:
set error to `'chrome.tabs.duplicate could not find tabId ' + tabId` - I haven't actually verified that message against Chrome, but that would be consistent with the others